### PR TITLE
Align method signature with interface

### PR DESCRIPTION
**Pull request by u01jmg3**
- `MenuItem::getLinkAttribute(string $name, $default = null)`
- `MenuItem::setChildrenAttribute(string $name, $value)`

### CODE REVIEW

File: src/Knp/Menu/MenuItem.php [hunk 1]
```diff
@@ -263,7 +263,7 @@ public function getChildrenAttribute(string $name, $default = null)
         return $default;
     }
 
-    public function setChildrenAttribute(string $name, $value): ItemInterface
+    public function setChildrenAttribute(string $name, string $value): ItemInterface
```
Comment (garak) [inline src/Knp/Menu/MenuItem.php]: It looks like casting `string` on attribute's value is too strong.
Indeed, the old code (see branch v2) suggests `string` in phpdoc, but likely the attribute's value could be something else.
So, please remove casting from class **and** from interface

Comment (u01jmg3) [reply src/Knp/Menu/MenuItem.php]: Changes made although I see `setLinkAttribute(string $name, string $value)` still has `$value` casted as a `string`.

**Comment by garak** [comment]
Thanks @u01jmg3

File: src/Knp/Menu/MenuItem.php (hunk 1 above)
Comment (garak) [reply src/Knp/Menu/MenuItem.php]: You're right, it should be removed as well.

Comment (u01jmg3) [reply src/Knp/Menu/MenuItem.php]: See PR #314

**Pull request merged**